### ISSUE TITLE
Saveable Templates

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Illablu/google-refine#74]
```
Custom templates have to get re-entered every time an export is performed. 
Would be convenient, to be able to save and restore them some way. 

Simplest way I guess would be to allow the user to specify the Prefix, Row 
Template, Row Suffix and Footer in a file with some agreed upon seperator 
between them. The export UI can ask the user for the location of the file and 
populate the text boxes. Or the location could be added to and read out of a 
config file somewhere.  

Using Gridworks 1.1 878
```

Original issue reported on code.google.com by `sharunsa...@gmail.com` on 16 Jun 2010 at 9:21

**Comment by GoogleCodeExporter** [comment]
Original comment by `iainsproat` on 16 Jun 2010 at 9:53
- Changed state: **Accepted**
- Added labels: **Component-Persistence**, **Type-Enhancement**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
Original comment by `dfhu...@gmail.com` on 16 Jun 2010 at 3:03

**Comment by GoogleCodeExporter** [comment]
Original comment by `dfhu...@gmail.com` on 18 Jul 2010 at 1:08
- Added labels: **Milestone-1.5**

**Comment by GoogleCodeExporter** [comment]
Original comment by `dfhu...@google.com` on 27 Sep 2010 at 10:02
- Added labels: **Milestone-2.0**

**Comment by GoogleCodeExporter** [comment]
```
Already fixed before r1326. Will be available in 2.0.
```

Original comment by `dfhu...@google.com` on 27 Sep 2010 at 10:19
- Changed state: **Fixed**
